packages/rules/src/mustSatisfyPeerDependencies.ts

```diff
@@ -0,0 +1,272 @@
+/*!
+ * Copyright 2020 Palantir Technologies, Inc.
+ *
+ * Licensed under the MIT license. See LICENSE file in the project root for details.
+ *
+ */
+
+import { Context, RuleModule } from "@monorepolint/core";
+import { mutateJson, PackageJson } from "@monorepolint/utils";
+import path from "path";
+import * as r from "runtypes";
+import { coerce } from "semver";
+import { IPackageDependencyGraphNode, PackageDependencyGraphService } from "./util/packageDependencyGraphService";
+
+const Options = r.Union(
```

Comment:
I considered adding options for whitelist / blacklist of peer dependencies to check (e.g. "only check react" or "don't bother checking redux"). This seems particularly useful for adopting the rule, since ignoring peer dependencies seems to be pretty mainstream.

If this is something we want, I think it might be best to do in a follow-up PR, to keep this from getting any larger

Reply:
i think adding an option to whitelist peer deps would be a must for us to use this rule, agree on adding as a FLUP!

Reply:
https://github.com/monorepolint/monorepolint/pull/118